lib/measured/measurable.rb

```diff
@@ -1,6 +1,26 @@
 class Measured::Measurable < Numeric
   include Measured::Arithmetic
 
+  PARSE_REGEX = /
+    \A         # beginning of input
+    \s*        # optionally any whitespace
+    (          # capture the value
+      -?       # number can be negative
+      \d+      # must have some digits
+      (?:      # do not capture
+        \.     # period to split fractional part
```

Comment:
I think commas are pervasive enough to support, but a larger question to ask is how far do we go with localization here. Can take that to a :octocat: issue.